frontend_lib/src/component/Tags/NewTagForm.styl

```diff
@@ -0,0 +1,79 @@
+@import '../../css/Variable.styl'
+
+.tagList__form
+  padding 15px
+  flex-direction column
+  height 100%
+  width 100%
+  &__title
+    font-size fontSizeMedium
+    margin-bottom 25px
+  &__tag
+    margin-bottom 10px
+  &__close
+    position absolute
+    right 20px
+  &__member
+    margin 0 0 20px 0
+    &__name
+      position relative
+      .name__label
+        margin 0 0 15px 0
+        font-weight bold
+      .name__input
+        margin 0 0 0 0
+      .name__adminmsg
+        margin 10px 0 10px 0
+        font-size fontSizeSmall
+      .autocomplete
+        position absolute
+        width 300px
+        background-color offWhite
+        border-radius 10px
+        border-width 1px
+        border-style solid
+        z-index 1
+        &__item
+          display flex
+          align-items center
+          cursor pointer
+          padding 5px 8px
+          &:hover
+            background-color: grey-hover
+          &:first-child
+            border-top-left-radius 10px
+            border-top-right-radius 10px
+          &:last-child
+            border-bottom-left-radius 10px
+            border-bottom-right-radius 10px
+          &__avatar
+            width 45px
+            height 45px
+            border-radius 50%
+          &__icon
+            font-size 28px
```

Comment:
Can you use a font size from Variable.styl?

Reply:
We decided not to do this now, because this size is related to the circled plus button which is also in the new member form. We need to rework this and the one in the new member form too.